src/main/java/org/broadinstitute/hellbender/engine/IntervalWalker.java

```diff
@@ -56,7 +54,7 @@ protected final void onStartup() {
     }
 
     @Override
-    public void traverse() {
+    public final void traverse() {
```

Comment:
As discussed above, don't make this final, but do add the comment "You should only override traverse() if you are writing a new walker base class in the engine package that extends this class. It is not meant to be overridden by tools outside of the engine package."

Reply:
Fixed.